package.json

```diff
@@ -60,24 +62,35 @@
     "wordcloud": "^1.0.5"
   },
   "devDependencies": {
-    "babel-core": "^6.18.2",
-    "babel-loader": "^6.2.7",
-    "babel-polyfill": "^6.16.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "babel-preset-stage-3": "^6.17.0",
-    "css-loader": "^0.25.0",
-    "eslint": "^3.10.1",
-    "eslint-config-airbnb": "^13.0.0",
-    "eslint-import-resolver-webpack": "^0.7.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^2.2.3",
-    "eslint-plugin-react": "^6.7.1",
-    "file-loader": "^0.9.0",
-    "json-loader": "^0.5.4",
+    "babel-core": "6.18.0",
```

Comment:
ahhhh ok that explains why there was a merge conflict. I didn't look closely enough. My bad!